underscore.js

```diff
@@ -696,6 +696,20 @@
     return range;
   };
 
+  // Split an **array** into several arrays containing **count** or less elements
+  // of initial array
+  _.chunk = function(array, count) {
+    count = count != null ? count : 1;
```

Comment:
When there's two cases, as in all conditional expressions, don't negate the test. Should be `count = count == null ? 1 : count`. Or we could avoid an unnecessary assignment and use `if (count == null) count = 1;`.